wallet/src/Ergvein/Wallet/Monad/Auth.hs

```diff
@@ -108,7 +122,7 @@ instance (MonadBaseConstr t m, MonadRetract t m, PlatformNatives) => MonadFrontB
   getPasswordSetEF = asks env'passSetEF
   {-# INLINE getPasswordSetEF #-}
   requestPasssword reqE = do
-    idE <- performEvent $ (liftIO randomIO) <$ reqE
+    idE <- performEvent $ (liftIO getRandom) <$ reqE
```

Comment:
Brackets around liftIO are redundant

Reply:
done in 7dc9b0c75cae355c7eef2ff1d285723d60d0684c